Sources/MeowVapor/MeowVaporError.swift

```diff
@@ -1,3 +1,3 @@
-enum MeowVaporError: Error {
+public enum MeowVaporError: Error {
     case modelInParameterNotFound
```

Comment:
Can we do anything to prevent people from `switch`-ing over this enum? If we add new error cases that becomes problematic. At least put some comments there, since we don't have `@_unknown` yet

Reply:
I added a case that should make this message quite clear 😉